silverback/settings.py

```diff
@@ -69,8 +72,13 @@ def get_broker(self) -> AsyncBroker:
             broker = broker_class()
 
         else:
-            # TODO: Not all brokers share a common arg signature.
-            broker = broker_class(self.BROKER_URI or None)
+            # NOTE: BROKER_URI to be deprecated in 0.6
+            if self.BROKER_URI:
+                broker = broker_class(self.BROKER_URI or None)
+            elif self.BROKER_KWARGS:
+                broker = broker_class(**self.BROKER_KWARGS)
```

Comment:
Technically this would always work, even for an empty dict

Reply:
Not sure what you mean.

```python
>>> x = {}
>>> bool(x)
False
```

Reply:
I meant:

```py
def fn():
    pass


fn(**{})
```